.github/workflows/integration_test_with_secrets.yaml

```diff
@@ -4,40 +4,15 @@ on:
   pull_request:
 
 jobs:
-  integration-test-with-secrets:
-    runs-on: ubuntu-latest
-    name: Integration Tests (require secrets)
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Setup operator environment
-        uses: charmed-kubernetes/actions-operator@main
-        with:
-          provider: microk8s
-
-      - name: Enable microk8s plugins
-        run: |
-          sudo microk8s enable hostpath-storage ingress registry
-          sudo microk8s kubectl -n kube-system rollout status -w deployment/hostpath-provisioner
-          sudo microk8s kubectl -n ingress rollout status -w daemonset.apps/nginx-ingress-microk8s-controller
-          sudo microk8s kubectl -n container-registry rollout status -w deployment/registry
-
-      - name: Install tox
-        run: python3 -m pip install tox
-
-      - name: Config iptables
-        run: |
-          sudo iptables -I DOCKER-USER  -j ACCEPT
-
-      - name: Build docker image
-        run: |
-          docker build -f discourse.Dockerfile -t localhost:32000/discourse:test .
-          docker push localhost:32000/discourse:test
-
-      - name: Run integration tests
-        run: |
-          tox -e integration -- \
-          -m requires_secrets \
-          --saml-email ${{ secrets.TEST_SAML_EMAIL }} \
-          --saml-password ${{ secrets.TEST_SAML_PASSWORD }} \
-          --discourse-image=localhost:32000/discourse:test
+  integration-tests-with-secrets:
```

Comment:
We can probably remove this wf and use the "integration test" one without the `-m` argument, right?

Reply:
Sadly we can't. There are two integ tests conflicting with each other right now. I need to make sure which one are and fix it but this will be for another PR. I'll create a ticket for that.